css/properties/background-attachment.json

```
@@ -125,12 +135,29 @@
               "opera_android": {
                 "version_added": "14"
               },
-              "safari": {
-                "version_added": false
-              },
-              "safari_ios": {
-                "version_added": "4.2"
-              },
+              "safari": [
+                {
+                  "version_added": "13",
+                  "partial_implementation": true,
+                  "notes": "<code>local</code> is recognized but has no effect due to [a bug](https://bugs.webkit.org/show_bug.cgi?id=219324)."
+                },
+                {
+                  "version_added": "5",
+                  "version_removed": "13"
+                }
+              ],
+              "safari_ios": [
+                {
+                  "version_added": "13",
+                  "partial_implementation": true,
+                  "notes": "<code>local</code> is recognized but has no effect due to [a bug](https://bugs.webkit.org/show_bug.cgi?id=219324)."
+                },
+                {
+                  "version_added": "4.2",
+                  "version_removed": "13",
+                  "notes": "<code>local</code> is only supported when <code>-webkit-overflow-scrolling: touch</code> is NOT used."
```

For symmetry with the other notes, I'd propose new text like this:

```suggestion
                  "notes": "If <code>-webkit-overflow-scrolling: touch</code> is set, then <code>local</code> has no effect."
```

---

done ✅